app/model.js

```diff
@@ -11,6 +11,33 @@ const fetchTopics = () =>
     })
 }
 
+
+const fetchComments = (params) =>
+{
+    if(/[0-9]+/g.test(params))
```

Comment:
Whilst this is working, you don't need this conditional logic in my opinion. If the provided article_id were not a number then PSQL would throw an error which would be sent to the catch block in the controller. The next function in that controller would then pass that error onto the error handling middleware. 

I can't see where you are rejecting with a 400 to pass that test?

Reply:
there is an else statement at the bottom on line 38